etc/docker-scripts/bootstrapper.py

```diff
@@ -52,6 +52,8 @@ def package_install():
             to_install_list.append(package+'=='+ver)
 
     if to_install_list:
+        if 'crio-python-user-lib-path' in input_params:
```

Comment:
can we make this more generic where it could be used for different use cases? Is this mounting a volume for the user home directory? something like `home-volume-path` or `user-volume-path`?

Reply:
updated vars to more generic naming